src/Module/Conversation/Network.php

```diff
@@ -296,11 +296,9 @@ protected static function parseRequest(array $parameters, array $get)
 			self::$selectedTab = $get['order'];
 		}
 
-		DI::pConfig()->set(local_user(), 'network.view', 'selected_tab', self::$selectedTab);
-
-		self::$star    = intval($get['star']      ?? 0);
-		self::$mention = intval($_GET['mention']      ?? 0);
-		self::$order   = in_array(self::$selectedTab, ['received', 'commented', 'created', 'uriid']) ? self::$selectedTab : 'commented';
+		self::$star    = intval($get['star'] ?? 0);
+		self::$mention = intval($_GET['mention'] ?? 0);
+		self::$order   = $get['order'] ?? 'commented';
```

Comment:
This is wrong, I removed the redirection that ensured the `order` GET parameter was always present so this will forget the user's choice when they choose the order and then come back to the network page later. We absolutely need to store and retrieve the user's preference now.

Reply:
Checking from `$selectedTab` is a problem, since that value could contain `mention` as well - which is no order.

I also removed the storing in the pconfig table. This can cause weird situations when you are working on different computers in parallel.

Reply:
The tab is now stored in a session variable.